src/common/rollbar.config.js

```diff
@@ -129,11 +132,19 @@ function updateRollbarPerson(session){
   });
 }
 
+function scrubDomNodes(scrubNames){
+  return node => {
+    const nameAttr = find(node.attributes, attr => attr.key === 'name');
+    return !!nameAttr && includes(scrubNames, nameAttr.value);
```

Comment:
What does `!!` do?

Reply:
Same as `Boolean(nameAttr)`. A cast essentially. Idk what is most readable. If nameAttr is falsy it will return false instead of undefined or whatever. Negating it turns it into a Boolean and doing it again unnegates it.

Reply:
Same as `Boolean(nameAttr)`. A cast essentially. Idk what is most readable. If nameAttr is falsy it will return false instead of undefined or whatever. Negating it turns it into a Boolean and doing it again unnegates it.

Reply:
Ah, interesting.